.vsts-dotnet.yml

```diff
@@ -0,0 +1,129 @@
+phases:
```

Comment:
@jaredpar @cston I'm surprised to see changes not related to nullable in this merge. Is this expected?

Reply:
Per discussion with Jard, I'll remove this file